msys/members/urls.py

```diff
@@ -26,7 +27,10 @@
     
     url(r'^cards/$', views.cards, name='cardList'),
     url(r'^cards/add/$', views.addCard, name='addCard'),
+    path('cards/login/', views.noCardCode, name='noCardCode'),
+    path('cards/check/', views.noCardCode, name='noCardCode'),
     url(r'^cards/check/(?P<card_rfid>[ A-Fa-f0-9]+)/$', views.checkCard, name='checkCard'),
+    path('cards/login/<slug:card_rfid>', views.loginCard, name='loginCard'),
```

Comment:
We are expecting a hex string so it might be best to use the same regex as for checkCard

Reply:
Alright!